### ISSUE TITLE
1.7.1 – Inconsistent object formatting in function

### ISSUE THREAD

**Issue by poteto** [prettier/prettier#2916]
**Prettier 1.7.1**
[Playground link](https://prettier.io/playground/#%7B%22content%22%3A%22let%20thing%20%3D%20%7B%5Cn%20%20foo%3A%20%5C%22foo%5C%22%2C%5Cn%20%20bar%3A%20%5C%22bar%5C%22%2C%5Cn%20%20baz%3A%20%5C%22baz%5C%22%2C%5Cn%20%20qux%3A%20%5C%22qux%5C%22%2C%5Cn%20%20abc%3A%20%5C%22abc%5C%22%2C%5Cn%20%20def%3A%20%5C%22def%5C%22%2C%5Cn%20%20ghi%3A%20%5C%22ghi%5C%22%5Cn%7D%3B%5Cn%5Cntest(%5C%22it%20...%5C%22%2C%20function(assert)%20%7B%5Cn%20%20let%20thing%20%3D%20%7B%5Cn%20%20%20%20foo%3A%20%5C%22foo%5C%22%2C%5Cn%20%20%20%20bar%3A%20%5C%22bar%5C%22%2C%5Cn%20%20%20%20baz%3A%20%5C%22baz%5C%22%2C%5Cn%20%20%20%20qux%3A%20%5C%22qux%5C%22%2C%5Cn%20%20%20%20abc%3A%20%5C%22abc%5C%22%2C%5Cn%20%20%20%20def%3A%20%5C%22def%5C%22%2C%5Cn%20%20%20%20ghi%3A%20%5C%22ghi%5C%22%5Cn%20%20%7D%3B%5Cn%7D)%3B%22%2C%22options%22%3A%7B%22ast%22%3Afalse%2C%22bracketSpacing%22%3Atrue%2C%22doc%22%3Afalse%2C%22jsxBracketSameLine%22%3Afalse%2C%22output2%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22printWidth%22%3A80%2C%22semi%22%3Atrue%2C%22singleQuote%22%3Afalse%2C%22tabWidth%22%3A2%2C%22trailingComma%22%3A%22none%22%2C%22useTabs%22%3Afalse%7D%7D)

**Input:**
```jsx
let thing = {
  foo: "foo",
  bar: "bar",
  baz: "baz",
  qux: "qux",
  abc: "abc",
  def: "def",
  ghi: "ghi"
};

test("it ...", function(assert) {
  let thing = {
    foo: "foo",
    bar: "bar",
    baz: "baz",
    qux: "qux",
    abc: "abc",
    def: "def",
    ghi: "ghi"
  };
});
```

**Output:**
```jsx
let thing = {
  foo: "foo",
  bar: "bar",
  baz: "baz",
  qux: "qux",
  abc: "abc",
  def: "def",
  ghi: "ghi"
};

test("it ...", function(assert) {
  let thing = { foo: "foo", bar: "bar", baz: "baz", qux: "qux", abc: "abc", def: "def", ghi: "ghi" };
});
```

**Expected behavior:**
Formatting for the object should be consistent.

**Comment by duailibe** [comment]
This is the same as #2909, #2910 and #2915

**Comment by poteto** [comment]
@duailibe Sorry, I didn't search correctly. I'll close this issue in favor of #2909

**Comment by suchipi** [comment]
Just released 1.7.2 where this is fixed. Sorry about that!
